Airports/Other/EGRO.txt

```diff
@@ -0,0 +1,3 @@
+Andrew oil rig
```

Comment:
```suggestion
Andrew Oil Rig
```
Capitalisation of Oil and Rig for consistency

Reply:
done